packages/yarnpkg-libzip/package.json

```diff
@@ -4,6 +4,16 @@
   "license": "BSD-2-Clause",
   "main": "./sources/sync.ts",
   "browser": "./sources/async.ts",
+  "exports": {
+    ".": {
+      "node": "./sources/sync.ts",
+      "browser": "./sources/async.ts",
+      "default": "./sources/sync.ts"
+    },
+    "./sync": "./sources/sync.ts",
+    "./async": "./sources/async.ts",
```

Comment:
The main reason I did it this way was to provide a way to override this if needed, ie if a build system for some reason did not pull the browser version, it would leave a way to ensure the async version is gotten. I guess if it is an issue for someone they could open an issue later to ask for help on getting that right.